WeatherTodayTests/Sources/Mocks/MockWeatherUseCase.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  MockWeatherUseCase.swift
+//  WeatherTodayTests
+//
+//  Created by Taher on 20/10/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+import RxCocoa
+
+
```

Comment:
```suggestion
```

Reply:
fixed in this [commit](https://github.com/nimblehq/ci-cd-sample-ios/pull/22/commits/3d65e1179992cc9c7442660fdb88d9d668fb923c)